### PR TITLE
feat: Add button to delete profile

### PR DESCRIPTION
**Pull request by GeckoEidechse**
This adds a button and corresponding logic to delete an existing profile

image

Spliced out from #444

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
@Alystrasz it's been a while, you wanna test/review the PR? ^^

**Comment by Alystrasz** [comment]
Yeah I'm kinda focusing on launcher PRs these days, but I still have some time for Flightcore :)

**Review by Alystrasz — DISMISSED** [review]
Apart from the style change, feature looks good!

File: src-vue/src/views/SettingsView.vue [hunk 1]
```diff
@@ -6,6 +6,23 @@
     >
         <el-table :data="availableProfiles" >
             <el-table-column prop="name" label="Name" />
+            <el-table-column align="right">
+              <template #default="scope">
+                <el-popconfirm
```
[the remark this answers is not in the document]
Comment (GeckoEidechse) [reply src-vue/src/views/SettingsView.vue:11]: So I just add like a

```css
overflow-wrap: break-word;
```
?

[the remark this answers is not in the document]
Comment (Alystrasz) [reply src-vue/src/views/SettingsView.vue:11]: That should do it yeah

[the remark this answers is not in the document]
Comment (GeckoEidechse) [reply src-vue/src/views/SettingsView.vue:11]: Given that the profile functionality is still behind dev flag I'm gonna turn this PR review comment into a new issue and merge as-is in order to not block development.

We should definitely fix it before proper public release of profiles though \^\^

**Pull request merged**